polaris.shopify.com/content/foundations/content/grammar-and-mechanics.md

```diff
@@ -1045,25 +930,12 @@ Don’t use colons to introduce radio buttons or checkboxes.
 
 #### Do
 
-If the customer abandons their checkout, send them an email reminder to complete
-their order
-
-<p>
-  <label><input type="radio" name="foo"> Never</label><br />
-  <label><input type="radio" name="foo"> 6 hours later</label><br />
-  <label><input type="radio" name="foo"> 24 hours later</label>
-</p>
+If the customer abandons their checkout, send them an email reminder to complete their order
 
 #### Don’t
 
-If the customer abandons their checkout, send them an email reminder to complete
-their order:
+If the customer abandons their checkout, send them an email reminder to complete their order:
```

Comment:
Out of scope for this PR, but in our docs, it's often hard to separate body text from example UI copy.